src/transformers/tokenization_utils_base.py

```diff
@@ -1342,8 +1342,9 @@ def all_special_ids(self) -> List[int]:
                 tokenizer assumes the input is already split into words (for instance, by splitting it on whitespace)
                 which it will tokenize. This is useful for NER or token classification.
             pad_to_multiple_of (`int`, *optional*):
-                If set will pad the sequence to a multiple of the provided value. This is especially useful to enable
-                the use of Tensor Cores on NVIDIA hardware with compute capability `>= 7.5` (Volta).
+                If set will pad the sequence to a multiple of the provided value. Requires `padding` to be set to
+                `True`. This is especially useful to enable the use of Tensor Cores on NVIDIA hardware with compute
```

Comment:
That's not entirely correct. Any value different from False or do_not_pad will trigger this.

Reply:
If I understand correctly, the other possible values are `longest` and `max_length`, which also control the length and will contradict the `pad_to_multiple_of` setting. In this case, it's not clear which length value will take priority. 

So while they may trigger this, this way probably shouldn't be recommended.

Reply:
`longest` and `True` are doing the same thing so both are valid. As for `max_length` it will error if the max length passed is not a multiple of this.

So I insist that this should read "requires padding to be activated" and not just "requires padding to be set to True" as the latter is incorrect.